src/placement/pl_map.c

```diff
@@ -314,8 +314,8 @@ static pthread_rwlock_t		pl_rwlock = PTHREAD_RWLOCK_INITIALIZER;
 /** hash table for placement maps */
 static struct d_hash_table	pl_htable;
 
-#define DSR_RING_DOMAIN         PO_COMP_TP_RACK
-#define DSR_JUMP_MAP_DOMAIN      PO_COMP_TP_RACK
+/** XXX should be fetched from property */
```

Comment:
should the XXX be removed or is this a to-do item that needs a ticket

Reply:
Yeah, this is a to-do item. I expect there's already a ticket, though I'm not sure who it belongs to. I copied some things I needed in this file directly from a patch Johann created, in hopes of avoiding conflicts for his piece landing afterward.